com.unity.render-pipelines.high-definition/Editor/Compositor/CompositorWindow.cs

```diff
@@ -12,10 +12,11 @@ namespace UnityEditor.Rendering.HighDefinition.Compositor
 {
     internal class CompositorWindow : EditorWindow
     {
-        static partial class Styles
+        static class Styles
         {
-            static public readonly GUIContent k_EnableCompositor = EditorGUIUtility.TrTextContent("Enable Compositor", "Enabled the compositor and creates a default composition profile.");
-            static public readonly GUIContent k_RemoveCompositor = EditorGUIUtility.TrTextContent("Remove compositor from scene", "Removes the compositor and any composition settings from the scene.");
+            static public GUIContent windowTitle { get; } = new GUIContent("Graphics Compositor");
```

Comment:
This still miss localization.
According to https://confluence.unity3d.com/pages/viewpage.action?spaceKey=LOC&title=Editor+Localization+Coding+Guidelines , windows title this should be localizable too. (There is no exception rules)